.github/ISSUE_TEMPLATE/v8_breaking_change.md

```diff
@@ -11,15 +11,15 @@ assignees: ''
 
 **Which release will ship the breaking change?**
 
-<!-- e.g., v7.6.2 -->
+8.0
```

Comment:
The issue template states that it's an 8.0 breaking change, so it feels like this will be 8.0 for all of these? Or perhaps we should change `Which release will ship the breaking change?` to `Which release first included the deprecation?`, but I'm not sure how the answer to this new question impacts anything.

Reply:
Good call hardcoding this in here. I don't think we'll ever encounter a scenario where we'll need to change this between now and the time 8.0 is GA.